README.md

```diff
@@ -130,11 +130,13 @@ Checkout the branch locally and run:
 
 `yarn test-visual:approve` - This will approve the failures/diff caught by the tests.
 
-`yarn test-visual:reference` - This will update the reference images locally on your machine.
+`yarn test-visual:reference` - This will update the reference images locally on your machine. Run this command if you've approved your reference images but the local tests are still failing.
 
 `git lfs push --all origin` - First commit the files in the normal way then run the command. This will push the new reference images to Git LFS.
 
-You can then commit and push your changes. The test images that would have been created when you ran `yest test-visual` are gitignored and the new references images will be pushed to Git LFS.
+If your local tests are failing but you have approved them, run `yarn test-visual:reference`. This will update the reference images locally on your machine.
+
+You can then commit and push the updated references and your changes to your branch. The test images that would have been created when you ran `yarn test-visual` are gitignored and the new references images will be pushed to Git LFS.
```

Comment:
```suggestion
You can then commit and push the updated references and your changes to your branch. The test images that would have been created when you ran `yarn test-visual` are gitignored and the new references images will be pushed to Git LFS.

If your local tests are failing but you have approved them, run `yarn test-visual:reference`. This will update the reference images locally on your machine.
```